step_01/ReadMe.md

```diff
@@ -13,10 +13,10 @@ A client link will be displayed in the console. Please cut & paste it into a web
 ```python
 from taipy import Gui
 
-Gui(page="# Getting started with *Taipy*").run()
+Gui(page="# Getting started with *Taipy*").run() #user_reloader=True
```

Comment:
You meant 'use_reloader', right?
A space  after the sharp sign will not hurt the reading